CMakeLists.txt

```diff
@@ -544,6 +536,12 @@ if(${VELOX_ENABLE_CLP_CONNECTOR})
   velox_set_source(spdlog)
   velox_resolve_dependency(spdlog)
 
+  velox_set_source(microsoft_gsl)
+  velox_resolve_dependency(microsoft_gsl)
```

Comment:
* Can we move this above `nlohmann_json` so they're alphabetical?
* Can we add comments to group all dependencies into "Independent dependencies" and "Dependencies which depend on other dependencies"?